lib/daru/dataframe.rb

```diff
@@ -1346,7 +1346,7 @@ def index= idx
     #   df.vectors = Daru::Index.new([:foo, :bar, :baz])
     #   df.vectors.to_a #=> [:foo, :bar, :baz]
     def vectors= idx
-      unless index.is_a?(Daru::Index)
+      unless idx.is_a?(Daru::Index)
```

Comment:
The above code is invalid. The RHS must be a `Daru::Index`.

I don't get this... the statement is `unless idx.is_a?(Daru::Index)`. That means that the error should be thrown iff the object passed is anything but a `Daru::Index`.